src/main/java/frc/robot/RobotContainer.java

```diff
@@ -11,7 +11,7 @@
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.RunCommand;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
-
+import frc.robot.Constants.TiltedElevatorConstants;
```

Comment:
In `RobotContainer`? Also I put a newline between the `edu.*` imports and our `frc.*` imports but vscode doesn't so I manually add them in afterwards